specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -128,60 +125,60 @@ should be created only when there is a natural developer constituency
 and a well-understood class of applications that can benefit from it. It
 is also recommended that a profile cast a comprehensive net on its area
 of interest, to minimize the occurrence of overlapping or competing
-profiles. Java EE platform features such as optional components and
+profiles. Jakarta EE platform features such as optional components and
 extensibility can be used by profiles to achieve a better fit to their
 intended target.
 
 === Application Components
 
-The Java EE runtime environment defines four
-application component types that a Java EE product must support:
+The Jakarta EE runtime environment defines four
+application component types that a Jakarta EE product must support:
 
 * Application clients are Java programming
 language programs that are typically GUI programs that execute on a
 desktop computer. Application clients offer a user experience similar to
 that of native applications and have access to all of the facilities of
-the Java EE middle tier.
+the Jakarta EE middle tier.
 * Applets are GUI components that typically
 execute in a web browser, but can execute in a variety of other
 applications or devices that support the applet programming model.
-Applets can be used to provide a powerful user interface for Java EE
+Applets can be used to provide a powerful user interface for Jakarta EE
 applications. (Simple HTML pages can also be used to provide a more
-limited user interface for Java EE applications.)
-* Servlets, JSP pages, JSF applications,
+limited user interface for Jakarta EE applications.)
+* Jakarta Servlets, Jakarta Server Pages pages, Jakarta Server Faces applications,
 filters, and web event listeners typically execute in a web container
-and may respond to HTTP requests from web clients. Servlets, JSP pages,
-JSF applications, and filters may be used to generate HTML pages that
+and may respond to HTTP requests from web clients. Jakarta Servlet, Jakarta Server Pages pages,
```

Comment:
"Servlets" here is a reference to the application components, not the spec.
It should not be changed to "Jakarta Servlet".

Reply:
@kwsutter In those cases the application components are "pages", e.g., JSP Pages, aka Jakarta Server Pages pages (yuck).

Reply:
Changed the Servlet reference.

> @kwsutter In those cases the application components are "pages", e.g., JSP Pages, aka Jakarta Server Pages pages (yuck).

But AFAIK, we can't use the JSP acronym on this context, right?

Reply:
Per the later comment, let's generalize this statement and use "Servlets, server pages, server faces, ...".  This approach is much more readable.